compiler/src/iree/compiler/Codegen/Common/test/combine_layout_transformation.mlir

```diff
@@ -155,19 +195,47 @@ func.func @fold_pack_op(%source : tensor<250x250xf32>, %result : memref<2x2x128x
 // CHECK-LABEL: @fold_pack_op
 //  CHECK-SAME:   %[[SOURCE:[a-zA-Z0-9_]+]]
 //  CHECK-SAME:   %[[RESULT:[a-zA-Z0-9_]+]]
 //   CHECK-DAG:   %[[TRUE:.+]] = arith.constant true
-//       CHECK:   %[[PAD:.+]] = tensor.pad %[[SOURCE]] low[0, 0] high[6, 6]
+//   CHECK-DAG:   %[[PAD_VAL:.+]] = arith.constant 0.000000e+00 : f32
+//   CHECK-DAG:   %[[C0:.+]] = arith.constant 0 : index
+//   CHECK-DAG:   %[[C1:.+]] = arith.constant 1 : index
+//   CHECK-DAG:   %[[C250:.+]] = arith.constant 250 : index
 //       CHECK:   %[[MAP_SCATTER_DEST:.+]] = tensor.empty() : tensor<2x2x128x128xf32>
 //       CHECK:   %[[MAP_SCATTER:.+]] = iree_linalg_ext.map_scatter
-//  CHECK-SAME:     %[[PAD]] into %[[MAP_SCATTER_DEST]] {
+//  CHECK-SAME:     %[[SOURCE]] into %[[MAP_SCATTER_DEST]] {
 //  CHECK-NEXT:   ^bb0(%[[IDX0:.+]]: index, %[[IDX1:.+]]: index):
 //       CHECK:     %[[DELINEARIZE0:.+]]:2 = affine.delinearize_index %[[IDX0]]
 //  CHECK-SAME:       into (2, 128) : index, index
 //       CHECK:     %[[DELINEARIZE1:.+]]:2 = affine.delinearize_index %[[IDX1]]
 //  CHECK-SAME:       into (2, 128) : index, index
 //       CHECK:     iree_linalg_ext.yield %[[DELINEARIZE0]]#0, %[[DELINEARIZE1]]#0, %[[DELINEARIZE0]]#1, %[[DELINEARIZE1]]#1, %[[TRUE]]
-//       CHECK:   } : tensor<256x256xf32> into tensor<2x2x128x128xf32> -> tensor<2x2x128x128xf32>
+//       CHECK:   } : tensor<250x250xf32> into tensor<2x2x128x128xf32> -> tensor<2x2x128x128xf32>
 //       CHECK:   iree_codegen.store_to_memref %[[MAP_SCATTER]], %[[RESULT]] : tensor<2x2x128x128xf32> into memref<2x2x128x128xf32>
+
+//       CHECK:   scf.forall (%[[WG_IV0:.+]], %[[WG_IV1:.+]]) = (0, 0) to (256, 256) step (1, 64) {
+//   CHECK-DAG:     %[[WG_TILE_UB0:.+]] = affine.min #[[$MAP]](%[[WG_IV0]])
+//   CHECK-DAG:     %[[WG_TILE_UB1:.+]] = affine.min #[[$MAP1]](%[[WG_IV1]])
+//       CHECK:     scf.for %[[IDX0:.+]] = %[[WG_IV0]] to %[[WG_TILE_UB0]] step %[[C1]] {
+//       CHECK:       scf.for %[[IDX1:.+]] = %[[WG_IV1]] to %[[WG_TILE_UB1]] step %[[C1]] {
+//   CHECK-DAG:         %[[EXPANDED_IDX0:.+]]:2 = affine.delinearize_index %[[IDX0]] into (2, 128) : index, index
+//   CHECK-DAG:         %[[EXPANDED_IDX1:.+]]:2 = affine.delinearize_index %[[IDX1]] into (2, 128) : index, index
+//   CHECK-DAG:         %[[IDX0_IS_LOW_PAD:.+]] = arith.cmpi ult, %[[IDX0]], %[[C0]] : index
+//   CHECK-DAG:         %[[IDX0_IS_HIGH_PAD:.+]] = arith.cmpi uge, %[[IDX0]], %[[C250]] : index
+//   CHECK-DAG:         %[[IDX0_IS_PAD:.+]] = arith.ori %[[IDX0_IS_LOW_PAD]], %[[IDX0_IS_HIGH_PAD]] : i1
+//   CHECK-DAG:         %[[IDX1_IS_LOW_PAD:.+]] = arith.cmpi ult, %[[IDX1]], %[[C0]] : index
+//   CHECK-DAG:         %[[IDX1_IS_HIGH_PAD:.+]] = arith.cmpi uge, %[[IDX1]], %[[C250]] : index
+//   CHECK-DAG:         %[[IDX1_IS_PAD:.+]] = arith.ori %[[IDX1_IS_LOW_PAD]], %[[IDX1_IS_HIGH_PAD]] : i1
+//   CHECK-DAG:         %[[IS_PAD:.+]] = arith.ori %[[IDX0_IS_PAD]], %[[IDX1_IS_PAD]] : i1
```

Comment:
nit: I'd drop all the types from the check because they don't provide values to me. (Same comment for the other new lit tests.)

> Tests should be minimal, and only check what is absolutely necessary.

https://mlir.llvm.org/getting_started/TestingGuide/#filecheck-best-practices